common/src/deployment.rs

```diff
@@ -32,6 +32,8 @@ pub enum EcsState {
     Stopped,
     Stopping,
     Failed,
+    /// Fallback
+    Unknown,
```

Comment:
Do we really want to introduce that ? When can it happen ?

Reply:
Made it a fallback when we are parsing an ECS string response into the enum. We don't cover states that we don't expect to see.